mobile/src/store/middlewares/handle-error/handle-error.middleware.ts

```diff
@@ -12,7 +13,10 @@ const handleError: Middleware =
   ({ dispatch }: HandleErrorParams) =>
   (next) =>
   (action) => {
-    if (action.error) {
+    if (
+      action.error &&
+      action.type !== `${ActionType.LOAD_CURRENT_USER}/${DataStatus.REJECTED}`
```

Comment:
Expected behavior - if error message is not 401 then show error, otherwise sign out and don't show error. We should handle this error in all requests, not only when we load current user. One way to achieve this is we have InvalidCredentialsError and CustomExceptionName in shared. In http service you can check 401 error status and then throw an InvalidCredentialsError. After that here you can check if error.name equals CustomExceptionName.INVALID_CREDENTIALS